docs/frontend/external-bus.md

```diff
@@ -155,3 +157,19 @@ Payload structure:
 
 }
 ```
+
+### Write Tag `tag/write`
+
+Available in: Home Assistant 0.115
+Type: `tag/write`
+Direction: frontend to external app
+Expect answer: no
+
+Tell the external app to open the UI to write to a tag. Name is the name of the tag as entered by the user. The name is `null` if no name has been set.
+
+```ts
+{
+  tag: string;
+  name: string | null;
```

Comment:
What would the purpose of the name field be?

Reply:
Show in the UI as confirmation that you tapped the right button. 

"Writing Creedence Clearwater Revival"